layouts/try/baseof.html

```diff
@@ -48,7 +48,7 @@ <h2>Interactive Tutorials</h1>
             <div class="col-lg-8 mr-2 ml-2">
                 <div class="row">
                     <div class="col-md-4 pl-1 pr-1 pb-3">
-                        <a href="https://katacoda.com/ncskier/scenarios/tekton-dashboard" target="_blank" rel="noopener">
+                        <a href="https://katacoda.com/tektoncd/scenarios/dashboard" target="_blank" rel="noopener">
```

Comment:
```suggestion
                        <a href="https://katacoda.com/tektoncd/scenarios/tekton-dashboard" target="_blank" rel="noopener">
```
nit: the url is incorrect.

Reply:
See my comment above in response to Andrea's question, this is an intentional change.